sourcepoint-ui/TCFv2_banner_amp.css

```
@@ -0,0 +1,443 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```

I would suggest making this a little more explicit.

```suggestion
https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
```